### ISSUE TITLE
Libcoral C++ on WSL2 Debian Bullseye

### ISSUE THREAD

**Issue by ArchitB22976** [google-coral/libcoral#29]
### Description

![image](https://user-images.githubusercontent.com/115623542/226167893-d96c61ac-a6c2-4a08-bf04-0fa6b3d9eca7.png)
I've recently made an attempt to run the libcoral examples on C++ with no success. This is the error message I get. Am I supposed to use an older version of Bazel? I'm using Bazelisk, if that matters.

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

_No response_

### Operating System

Linux

### Coral Device

USB Accelerator

### Programming Language

C++

### Relevant Log Output

_No response_</details>

**Comment by hjonnala** [comment]
@ArchitB22976 Yes, please try with bazel version 4.0.0 : https://github.com/google-coral/libcoral/blob/master/docker/Dockerfile#L81

**Comment by ArchitB22976** [comment]
Thanks. I've gotten rid of this set of errors, but now I've got a new set of errors. Please have a look.
![image](https://user-images.githubusercontent.com/115623542/227866240-8d14e88b-288e-495d-b07a-ef135fe06971.png)

**Comment by ArchitB22976** [comment]
Hi, I got rid of the errors, only to get a new set of errors.
![image](https://user-images.githubusercontent.com/115623542/227866682-83eff56c-fdaa-4eba-96e6-16386bdf090f.png)

**Comment by ArchitB22976** [comment]
Hi, I've tried using the 4.0 version. While it does solve the error messages I was getting in this issue, newer problems have cropped up, which looks like it's a problem with trying to run the inference program on WSL2. I'm linking back to issue  https://github.com/google-coral/libcoral/issues/13
because from what I can tell, my current set of error messages are the same as the OP in that issue. As noted there, there is no support for WSL. 

In case someone stumbles across this issue, there you go. I may have misled @hjonnala here with my tags (certainly not intentionally), because I was trying to get the closest possible tags related to my issue, but otherwise this issue is unresolved for the time being. In case anyone is wondering, I've made an attempt to run this program on a Debian 11.4 VM on VMWare player, and I get the same set of error messages as with WSL. I'm assuming that there isn't any support for virtual instances of Linux distros. I was trying to run these virtual instances because I initially was trying to run this program on Windows, but quickly came to realise that it was going to be too much effort trying to make that possible, and my work PC doesn't have dual boot, so Linux was not an option there. I do have intentions to run dual boot on my home PC (for a completely different reason), so I guess this just makes it easier for me to prioritise that and make another attempt.

All that said, the current solution, outside of trying to run the program on a Linux distro directly (not virtually), is to run an instance on Docker, since the provided Dockerfile abstracts a lot of the dependency management required to run the program. While I appreciate that, I'm trying to avoid using Docker, since my line of work requires working as closely with the hardware related to the program as possible. I know, using the VMWare for this was very counter-intuitive, but I was getting fairly desperate and was just trying to get a working instance running the program, but otherwise WSL was what I figured would've been a better option to run Linux virtually.

I'm really only writing all this to make sure that anyone else in a similar situation can either avoid using this repo entirely (if they can get away with using the Python version of this library and are stuck with Windows) or brace themselves to using Docker or a Linux machine (if they're working closely with the hardware). Outside of that, there is (in my experience) no support for any other options when it comes to running this library. Not sure why it's this restrictive, considering that the USB accelerator should be targetting hobbyists, but it seems that the general assumption is that the kind of people going for the device would be using Linux if they want to run the C++ version. I'm fairly new to all this in general, so if I've missed anything out, it's largely because most of what I've experienced here is based on the materials I could find online to try and get this thing running. If there's anyone who's managed to run instances of this repo on a Windows machine (WSL, VMWare or otherwise) sucessfully, I'd like to welcome them to share how they've done it here. Obviously, if you've used Docker, my understanding is that the docker build provided here emulates the hardware the program is to run on for compilation and execution, and that's not exactly what I'm looking for. If my understanding is incorrect, I'll appreciate any corrections on that front.

Leaving this issue open so that @hjonnala has an opportunity to add on to this if possible. Please feel free to close this if you think it's better to close it, but I'd like it if you could elaborate further (or correct me if I misunderstood something) on the usage and such.
